src/Workspaces/VisualBasic/Portable/Recommendations/VisualBasicRecommendationServiceRunner.vb

```diff
@@ -53,6 +54,29 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Recommendations
             Return ImmutableArray(Of ISymbol).Empty
         End Function
 
+        Protected Overrides Function IsInvocationExpression(syntaxNode As SyntaxNode) As Boolean
+
+            Return syntaxNode.IsKind(SyntaxKind.InvocationExpression)
+        End Function
+
+        Protected Overrides Function IsLambdaExpression(syntaxNode As SyntaxNode) As Boolean
+
+            Return TypeOf syntaxNode Is LambdaExpressionSyntax
+        End Function
+
+        Protected Overrides Function TryGetOrdinalInArgumentList(argumentOpt As SyntaxNode, ByRef ordinalInInvocation As Integer) As Boolean
+
+            Dim argument = TryCast(argumentOpt, ArgumentSyntax)
+            Dim argumentList = TryCast(argumentOpt.Parent, ArgumentListSyntax)
+            If argument IsNot Nothing AndAlso argumentList IsNot Nothing Then
+                ordinalInInvocation = argumentList.Arguments.IndexOf(argument)
+                Return True
+            End If
+
+            ordinalInInvocation = -1
+            Return False
+        End Function
```

Comment:
so, fwiw, i think you could use ISyntaxFacts and not need the top two methods.  You could also move this method into the base type.